Tests/Public/Core/DatabaseQueue/DatabaseQueueTests.swift

```diff
@@ -28,4 +28,154 @@ class DatabaseQueueTests: GRDBTestCase {
             }
         }
     }
+    
+    func testReleaseMemory() {
+        // TODO: test DatabaseQueue.releaseMemory() ?
```

Comment:
What I care about is that sqlite3_db_release_memory has been called when releaseMemory returns.

We could take inspiration from https://github.com/groue/GRDB.swift/blob/v0.70.1/Tests/Private/DatabasePoolReleaseMemoryTests.swift#L16 and add to Configuration a closure to be called in Database.releaseMemory(). Those tests would fo in the Private directory, because this closure would not be exposed in the public API.

Until then, I think this testReleaseMemory() method should be removed. The TODO is already written at https://github.com/groue/GRDB.swift/blob/v0.70.1/GRDB/Core/DatabasePool.swift#L135